hazelcast/src/main/java/com/hazelcast/map/impl/MapService.java

```diff
@@ -19,24 +19,13 @@
 import com.hazelcast.core.DistributedObject;
```

Comment:
Why do we need special support for MapService for quorum? Isn't something provided for all operations?